packs/trello/config.yaml

```diff
@@ -3,7 +3,16 @@
   token: ""
 
   list_actions_sensor:
-    board_id: c39TEFLt
-    list_id: 55e7456df81e21e9b4aea429
-    filter:
-      - createCard
+    #api_key: ""
+    #token: ""
+    lists:
```

Comment:
Unclear question about monitoring several GitHub repositories/several independent Trello Lists in one Sensor with big shared config: 
https://github.com/StackStorm/st2/pull/1739#issuecomment-140725275